src/main/java/uk/gov/ons/ctp/integration/eqlaunch/service/EqLaunchService.java

```diff
@@ -19,7 +19,8 @@ String getEqLaunchJwe(
       String formType,
       String accountServiceUrl,
       String accountServiceLogoutUrl,
-      KeyStore keyStore)
+      KeyStore keyStore,
```

Comment:
I think these methods have far too many parameters and we should create some kind of EqLaunchDTO object to pass in.  Perhaps we can get Phil to agree to this?

Reply:
In principle I agree with Rob. But we have two launch methods, with common fields and method specific fields. We could have a common launch DTO? 

Then the methods in question would become :

  public String getEqLaunchJwe(
      LaunchRequestDTO launchRequestDTO,
      CaseContainerDTO caseContainer,
      String userId,
      String accountServiceUrl,
      String accountServiceLogoutUrl)

Marginally better... my problem with builders is that a dev does not necessarily see all the fields when writing the code. Calling a method that demands 12 specific params is in your face.
If we had a DTO that encompassed the superset of the params from both methods, it is really not clear exactly which fields are required by each method.

At least as is, each method makes it clear exactly what is required to call it. 
I think we will leave as is.

Reply:
Don't agree, but I defer to Phil

Reply:
If you feel strongly, Rob, please provide an example of what the signature to these two methods would become.

Reply:
Your separate branch looks ok Rob. 

Perhaps the EqLaunchData could be extended to cover the additional fields that the second method requires?
Then that fatter DTO could also be passed to the method creating the payload?

Reply:
Done change to my branch to add another DTO object. Awaiting agreement to go forward